dbt_project.yml

```diff
@@ -28,6 +28,12 @@ models:
     ens:
       +schema: ens
       +materialized: view
+    airdrop:
+      +schema: airdrop
+      +materialized: view
```

Comment:
This and below should be `table`?

Reply:
@aalan3 oh right, I just enforced it directly in the model but you're right, it's safer to have it by default here too

Reply:
all good now